vyper/cli/vyper_json.py

```diff
@@ -262,6 +262,7 @@ def compile_from_input_dict(
     evm_version = get_evm_version(input_dict)
 
     optimize = input_dict["settings"].get("optimize")
+    experimental_codegen = input_dict["settings"].get("experimentalCodegen", False)
```

Comment:
hmm this is a new feature right?

Reply:
yes, it was necessary to actually test the json properly.
do we need documentation updates btw?

Reply:
idk, seems out of scope. we should have another PR for it i think